pkgs/games/cataclysm-dda/git.nix

```diff
@@ -22,24 +22,6 @@ let
       inherit rev sha256;
     };
 
-    patches = [
-      # Unconditionally look for translation files in $out/share/locale
-      ./locale-path.patch
```

Comment:
Not sure this one is safe to remove. If the Nixpkgs Darwin+tiles build still puts translation files in `share/locale` (by running `make install` during install phase), then we'll need that patch for the resulting binary to pick the translation files up.

Reply:
ah, looks like i confused translations.cpp with translation.cpp, thought the corresponding lines were removed completely. my bad

Reply:
```
diff --git a/src/translations.cpp b/src/translations.cpp
--- a/src/translations.cpp
+++ b/src/translations.cpp
@@ -52,13 +52,11 @@ std::string locale_dir()
 #define CATA_IS_ON_BSD
 #endif

-#if !defined(__ANDROID__) && ((defined(__linux__) || defined(CATA_IS_ON_BSD) || (defined(MACOSX) && !defined(TILES))))
     if( !PATH_INFO::base_path().empty() ) {
         loc_dir = PATH_INFO::base_path() + "share/locale";
     } else {
         loc_dir = PATH_INFO::langdir();
     }
-#endif
 #endif // LOCALIZE
     return loc_dir;
 }
```

Patch for the git version of the package.